packages/terra-alert/src/Alert.module.scss

```diff
@@ -55,6 +55,7 @@
   }
 
   .section {
+    display: flex;
```

Comment:
This causes the layout of the section contents to break and not display as inline any longer (the title line breaks and wraps, inner blocks layout horizontally as flexes which in not intended). Removing will correct the layout.